frontend/src/BlogList/BlogList.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import PaginationList from './PaginationList';
+import NavBar from '../Navbar/NavBar';
+import BlogContainer from './BlogContainer';
+import './BlogList.css';
```

Comment:
Separate the imports for created and packages module.
```javascript
import React from 'react';

import './BlogList.css';
import PaginationList from './PaginationList';
import NavBar from '../Navbar/NavBar';
import BlogContainer from './BlogContainer';
```

Reply:
Done po